views/doctor.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+from schemas import educations_schema, education_schema, doctors_schema, doctor_schema
+from flask import Blueprint, jsonify, request, Response
+from models.doctor import Doctor, Education
+from config import db
+from services.auth import token_required, get_user_from_request
+from services.doctor import get_doctor_by_id,\
+     is_equal_user_from_request_with_user_db        
+from services.services import get_object_by_id
+
+
+doctor_view = Blueprint("doctor", __name__)
+
+
+@doctor_view.route("/doctors", methods=["Get"])
+def get_doctors():
+    return jsonify(doctors_schema.dump(Doctor.query.all()))
+
+@token_required
+@doctor_view.route("/doctor", methods=["Post"])
+def create_doctor():
+    user = get_user_from_request(request)
+    doctor_data = doctor_schema.load(request.json)
+    new_doctor = Doctor(**doctor_data)
+    if user.doctor:
+        return Response("{'message':'user already have a doctor'}", status=400, mimetype='application/json')     
+    else:
+        user.doctor = new_doctor
+    db.session.add(new_doctor)
+    db.session.commit()
+
+    return Response(doctor_schema.dumps(new_doctor), status=201, mimetype='application/json')
+
+
+@doctor_view.route("/doctor/<id>", methods=["Get"])
+def get_doctor(id):
+    doctor = get_doctor_by_id(id)
+    return doctor_schema.dump(doctor)
+
+@token_required
+@doctor_view.route("/doctor/<id>", methods=["Put"])
+def put_doctor(id):
+    user = get_user_from_request(request)
```

Comment:
get_user_from_request looks like a good use case for decorator, so the user is just available in kwargs if you apply it like 
@get_user_from_request
def put_doctor(id, user):
...